app.rb

```diff
@@ -770,4 +771,19 @@ class App < Sinatra::Base
     erb :about
   end
 
+  get '/queue' do
+    @workers = Sidekiq::Workers.new.map do |_p, _t, worker|
```

Comment:
Use `_process_id` and `_thread_id` so it's obvious what those arguments are for, even though we currently are not using them.